Adding apogee paths to dr19

havok2063 opened the pull request:
This PR closes #104 and adds the apogee paths from ipl3 config to dr19 config.

joelbrownstein commented:
Hey Brian,

How did you add the abstract apogee drp paths to dr19.cfg? 

There are obvious mistakes such as references to APOGEE_ASPCAP since the dr19 version of the apogee drp does not include aspcap; and then environmental variable does not exist in the dr19.cfg of tree.

You know that I am not one of the apogee drp pipeline developers, so I'm not sure what you're expecting from my review of this PR?
 
 -Joel

havok2063 commented:
I just copied them from ipl-3.   I added Adam Wheeler and David Nidever to the review.  As lead data person, I added you for full completeness and transparency.

havok2063 commented:
@dnidever @ajwheeler @lyx12311 can someone look at the apogee paths for DR19, in the `dr19.cfg` and fix or remove erroneous paths that are incorrect or shouldn't be there.

dnidever commented:
Looks fine to me.  We don't need the APOGEE_ASPCAP or APOGEE_TARGET ones anymore, but it shouldn't break anything to have them in there.
